docs/source/releasenotes.rst

```diff
@@ -96,7 +96,6 @@ Release notes
   compatibility mode. (robot example: https://github.com/robocorp/example-ie-mode-edge)
 - Library **RPA.SAP**: Fix error in dependency import order in the underlying
   `SapGuiLibrary`.
->>>>>>> master
```

Comment:
This already got solved in the latest _master_.

Reply:
shouldn't be an issue with it being here then 😄